src/Array2D.luna

```diff
@@ -0,0 +1,235 @@
+import Std.Foreign.C.Value
+import Std.Foreign
+
+import Dataframes.Utils
+import Dataframes.Types
+
+class Size:
+    SizeVal:
+        rows   :: Int
+        columns:: Int
+
+    def getRows: case self of
+        SizeVal rows _: rows
+
+    def getColumns: case self of
+        SizeVal _ columns: columns
+
+class Stride:
+    StrideVal:
+        strideR:: Int
+        strideC:: Int
+
+    def getStrideR: case self of
+        StrideVal strideR _: strideR
+
+    def getStrideC: case self of
+        StrideVal _ strideC: strideC
+
+class Array2D a:
```

Comment:
(!) Please please please pretty please add some docstrings. We don't declare types yet, so the users will complain.